testeng/jobs/upgradePythonRequirements.groovy

```diff
@@ -28,7 +28,6 @@ def cronOffHoursBusinessDailyLahore = 'H H(0-5) * * 1-5'
 Map apiDocTools = [
     org: 'edx',
     repoName: 'api-doc-tools',
-    targetBranch: "master",
```

Comment:
These were all added in https://github.com/edx/jenkins-job-dsl/pull/1025 as part of preparation to support renaming the default branch in many repos to something like "main" instead of "master".  Could we retain the ability to specify this (maybe renamed to "defaultBranch"), but override it with the new parameter if provided?  The parameter should default to empty, since different repos will soon have different default branch names that we want to fall back to.

Reply:
My understanding was that the automated job will always run for the `master` branch of a repo so it should be a common default value for all the repos which can be renamed later to `main` if required. 
If different repos are going to have different default branches for the job then I'll add a check in the script as you suggested.

Reply:
dbt-schema-builder has already switched to `main` as the default branch: https://github.com/edx/dbt-schema-builder , and the plan is to switch more one-by-one (although there isn't a schedule for doing so yet).  This came out of the "Juneteenth" mini-hackathon: https://openedx.atlassian.net/wiki/spaces/ENG/pages/1553695423/How+To+Make+Engineering+More+Inclusive , and is related to a similar GitHub effort: https://www.cnet.com/news/microsofts-github-is-removing-coding-terms-like-master-and-slave/ .

I added a `github.default_branch` field to the repo health dashboard to track the status of this across our repos: https://docs.google.com/spreadsheets/d/1VCxNVq-niT-uv5BFmsYPF21r6I2-IQ-GJbidF0zUPBc/edit#gid=921158295